PendingReleaseNotes

```diff
@@ -3,8 +3,8 @@
 * New bluestore_rocksdb_options_annex config parameter. Complements
   bluestore_rocksdb_options and allows setting rocksdb options without repeating
   the existing defaults.
-* The cephfs addes two new CDentry tags, 'I' --> 'i' and 'L' --> 'l', and
-  on-RADOS metadata is no longer backwards compatible after upgraded to Pacific
+* The cephfs adds two new CDentry tags, 'I' --> 'i' and 'L' --> 'l', and
```

Comment:
s/The //

this one seems a bit in the weeds for a user @batrick

Reply:
I've removed "The".